src/ProophServiceProvider.php

```diff
@@ -94,4 +104,21 @@ public function provides()
             \Prooph\EventStore\Pdo\MySqlEventStore::class,
         ];
     }
+
+    /**
+     * Returns existing migration file if found, else uses the current timestamp.
+     *
+     * @param Filesystem $filesystem
+     * @return string
+     */
+    protected function getMigrationFileName(Filesystem $filesystem, string $filename): string
+    {
+        $timestamp = date('Y_m_d_His');
+
+        return Collection::make($this->app->databasePath().DIRECTORY_SEPARATOR.'migrations'.DIRECTORY_SEPARATOR)
```

Comment:
`DIRECTORY_SEPARATOR` here could just be slash. For example `return Collection::make($this->app->databasePath().'/migrations/')`.